dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityCollectionItemParentIT.java

```diff
@@ -0,0 +1,374 @@
+    Collection colAA2;
+    Collection colAB1;
+
+    Item itemAA1;
+    Item itemAA1MappedInAA2;
+    Item itemAA2;
+
+
+    @Before
+    public void setup() throws SQLException, AuthorizeException {
+        //We turn off the authorization system in order to create the structure as defined below
+        context.turnOffAuthorisationSystem();
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and one collection.
+        communityA = CommunityBuilder.createCommunity(context)
+                                     .withName("Parent CommunityA")
+                                     .build();
+        communityB = CommunityBuilder.createCommunity(context)
+                                     .withName("Parent CommunityB")
+                                     .build();
+        communityAA = CommunityBuilder.createSubCommunity(context, communityA)
+                                      .withName("Sub Community")
+                                      .build();
+        communityAB = CommunityBuilder.createSubCommunity(context, communityA)
+                                      .withName("Sub Community Two")
+                                      .build();
+        colAA1 = CollectionBuilder.createCollection(context, communityAA).withName("Collection 1").build();
+        colAA2 = CollectionBuilder.createCollection(context, communityAA).withName("Collection 2").build();
+        colAB1 = CollectionBuilder.createCollection(context, communityAB).withName("Collection 3").build();
+        communityService.addCollection(context, communityAB, colAA2);
+
+
+        itemAA1 = ItemBuilder.createItem(context, colAA1)
+                             .withTitle("Public item 1")
+                             .withIssueDate("2017-10-17")
+                             .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                             .withSubject("ExtraEntry")
+                             .build();
+
+        itemAA1MappedInAA2 = ItemBuilder.createItem(context, colAA1)
+                                        .withTitle("Public item 1")
+                                        .withIssueDate("2017-10-17")
+                                        .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                                        .withSubject("ExtraEntry")
+                                        .build();
+        collectionService.addItem(context, colAA2, itemAA1MappedInAA2);
+        itemAA2 = ItemBuilder.createItem(context, colAA2)
+                             .withTitle("Public item 1")
+                             .withIssueDate("2017-10-17")
+                             .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                             .withSubject("ExtraEntry")
+                             .build();
+
+        context.restoreAuthSystemState();
+    }
+
+    @Test
+    public void itemXOwningCollectionTest() throws Exception {
```

Comment:
can you rename the test methods according to the refactored name of the items variable?